migrator/clone/postgres/db_clone_manager.go

```diff
@@ -21,6 +21,9 @@ const (
 
 	// TempClone - temp postgres clone
 	TempClone = "central_temp"
+
+	// RestoreFromRocksClone - special clone for restoring from RocksDB
+	RestoreFromRocksClone = "central_rocks"
```

Comment:
Why do you start a new Clone name? Can we reuse RestoreClone? They shall not exist at the same time. If we enforce this rule in restore handler and anywhere else, it could be simpler in logic.

Reply:
We have logs and .backup to help us differentiate this situation. The good thing for reuse is: 1) we do not need to maintain RestoreFromRocksClone, like remove the residue from earlier failed operations. 2) We do not need to make sure there is at most one, RestoreFromRocksClone or RestoreClone at any time in a system.
I feel this is easier for me. If you believe this one is better for you, please double check and make sure ^.

Reply:
my concern is for the case where it fails in the middle and migration restarts.

postgres->postgres we have a central_restore from a dump that is complete though it may need postgres migrations.  If we fail and have a central_restore in that case, it will pick up that we have central_restore and process it.  We would have to put in logic in there to figure out if we need to find the RocksDB to migrate further.  Whereas my approach I use a separate DB, ignore it on the scan, and basically throw it away if something happens during the migration and start them over.

with your approach, we could pick up where we left off in theory.  I will think about how to move the logic to reuse it.

Reply:
I took a look and your approach does simplify things.  I have implemented that.